business_requirement_deliverable_cost/models/business.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirementResource(models.Model):
+    _inherit = "business.requirement.resource"
+
+    sale_price_unit = fields.Float(
+        string='Sales Price',
+        groups='business_requirement_deliverable_cost.'
```

Comment:
I guess this is a question more for functional user. These fields has sensible information and for our company is necessary to have different users levels to have restrictions on field level.

@elicoidal?